fenix/app/src/main/res/values/strings.xml

```diff
@@ -2101,7 +2101,7 @@
     <string name="radio_preference_info_content_description">Click for more details</string>
 
     <!-- Content description for the action bar "up" button -->
-    <string name="action_bar_up_description">Navigate up</string>
+    <string name="action_bar_up_description" moz:removedIn="124" tools:ignore="UnusedResources">Navigate up</string>
```

Comment:
That's a bit odd, not sure why this string was duplicated.

Reply:
It's not duplicated, it only added the attributes so that it can be removed later?

Reply:
The same ID is referenced in two different `strings.xml` files